.github/workflows/ci.yml

```diff
@@ -0,0 +1,115 @@
+name: CI
+on: [push, pull_request]
+
+jobs:
+  Test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [macos-latest, windows-latest]
+        node: [10, 12, 14, 15]
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node }}
+    - name: Install dependencies
+      run: npm install
+    - name: Unit tests
+      run: npm run test-unit
+    - name: Prebuild
+      if: matrix.node == 10 && env.prebuild_upload != '' && startsWith(github.ref, 'refs/tags/')
```

Comment:
We probably don't need to check for the `prebuild_upload` token, if we already check for the node version and tag.